tests/visualization_tests/matplotlib_tests/test_optimization_history.py

```diff
@@ -203,3 +185,24 @@ def fail_objective(_: Trial) -> float:
 
     figure = plot_optimization_history(studies, error_bar=True)
     assert len(figure.get_lines()) == 0
+
+
+@pytest.mark.parametrize("direction", ["minimize", "maximize"])
+def test_error_bar_at_optimization_history(direction: str) -> None:
```

Comment:
probably "in" not "at".
```suggestion
def test_error_bar_in_optimization_history(direction: str) -> None:
```